src/main/webapp/script.js

```diff
@@ -14,9 +14,12 @@
 
 const alertMessage = 'Sorry! We cannot geolocate you. Please enter a zipcode';
 let map;
+let locationQuery = '';
+let product = '';
```

Comment:
I might change this to productQuery, to keep it consistent, unless there is another reason for this difference.

Reply:
Oh wait I now see the reason, disregard haha